samples/Config.Options.Snapshot.Web/Startup.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+/// <summary>
+/// This sample demonstrates how IOptionsSnapshot will be created per request.
```

Comment:
/// This sample demonstrates an IOptionsSnapshot being created on the first request
/// after config.json changes. TimeOptions is bound to config.json, but TimeOptions
/// won't be recreated unless the
/// config file changes.  Hit the server and verify the creation time is
/// unchanged between requests.  Modify config.json and the time will
/// be updated on the next request.